Composer/packages/adaptive-form/src/components/withTypeIcons.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Is this a React component?  If so, casing of the withTypeIcons.tsx should be WithTypeIcons.tsx

Reply:
it's an hoc